docs/conflicts.md

```diff
@@ -3,15 +3,14 @@
 
 ## Introduction
 
-Like [Pijul](https://pijul.org/) and [Darcs](http://darcs.net/) but unlike most
-other VCSs, Jujutsu can record conflicted states in commits. For example, if you
-rebase a commit and it results in a conflict, the conflict will be recorded in
-the rebased commit and the rebase operation will succeed. You can then resolve
-the conflict whenever you want. Conflicted states can be further rebased,
-merged, or backed out. Note that what's stored in the commit is a logical
-representation of the conflict, not conflict *markers*; rebasing a conflict
-doesn't result in a nested conflict markers (see
-[technical doc](technical/conflicts.md) for how this works).
+Like [Darcs](http://darcs.net/) but unlike most other VCSs, Jujutsu can record
```

Comment:
Is it worth removing this direct comparison with Darcs in the first line of the document? I'm guessing this is where some might be confused because it says "like Darcs", which might indicate exact (or near-exact compatibility). We can perhaps add a reference to it at the end of the paragraph, or in further sections, with additional notes on the differences between Jujutsu and Darcs.

Reply:
That's fair. I suppose there's a risk that people think that Darcs models it in the same way. I don't think we need to mention it at all, so I just removed it.